ewatercycle/util.py

```diff
@@ -169,3 +171,16 @@ def data_files_from_recipe_output(recipe_output: RecipeOutput) -> Tuple[str, Dic
     # TODO simplify (recipe_output.location) when next esmvalcore release is made
     directory = str(data_files[0].filename.parent)
     return directory, forcing_files
+
+
+def to_absolute_path(input_path: str, parent: Path = None, must_exist: bool = False) -> Path:
```

Comment:
The current code does not use `must_exist=True`, can must_exist argument be dropped?

Reply:
I think there are useful cases for it, but we didn't check it before. This adds an easy way to make the package more robust.

Reply:
Ok, lets keep it, at least we have a test that excercises the argument.